composer.json

```diff
@@ -59,8 +59,25 @@
 		],
 		"check-complete-strict": [
 			"@php ./vendor/phpcsstandards/phpcsdevtools/bin/phpcs-check-feature-completeness ./WordPress"
+		],
+		"test": [
+			"@lint",
+			"@check-cs",
+			"@run-tests",
+			"@check-complete-strict"
 		]
 	},
+	"scripts-descriptions": {
+		"lint": "Lint PHP files against parse errors.",
+		"lint-ci": "Lint PHP files against parse errors, but output the results as Checkstyle XML (used for CI purposes).",
```

Comment:
Do we actually need the separate `lint-ci` script ?


In CI, we could just do `composer lint -- --checkstyle` to achive the same, so is a separate script necessary ?

Reply:
Agreed on this change.